DoxygenHelp.md

```diff
@@ -18,25 +18,31 @@ Alternatively install doxygen and dot yourself:
 
 ### Top-level CMakelists.txt
 If DOXYGEN_GENERATE_DOCUMENTATION is selected in the CMake GUI, CMake will automatically find the doxygen and dot executables. The up-to-date version of the file can be found here:
+
 https://github.com/tumcms/Open-Infra-Platform/blob/development/CMakeLists.txt
 
+In the CMake GUI, the following options can be selected:
+- DOXYGEN_OPTIONAL_COMMENTED_ONLY: If this is not selected, doxygen will assume that all entities are documented, even if there is no documentation available. 
+- DOXYGEN_OPTIONAL_INCLUDE_INTERNAL: If selected, this will include all comments preceded by "\internal" (comments intended for developers only).
+- DOXYGEN_OPTIONAL_INCLUDE_EARLYBINDING: If selected (... and already built), this will include the selected IFC schemas in the documentation. 
+
 ### ConfigureDoxygen.cmake file
 If DOXYGEN_GENERATE_DOCUMENTATION is selected in CMake, CMake will include ConfigureDoxygen.cmake. This file includes all settings for the generation of doxygen documentation for the Open Infra Platform project. Changes to documentation settings should be made here. doxygen_add_docs() creates the GenerateDocumentation target. The up-to-date version of the file can be found here:
 
 https://github.com/tumcms/Open-Infra-Platform/blob/development/cmake/ConfigureDoxygen.cmake
 
-Tags that are more likely to be required to change:
-- DOXYGEN_EXCLUDE: if you want to exclude certain source code directories or files from the documentation, you should add them here.
-- DOXYGEN_INTERNAL_DOCS: set to yes if you want to include comments included for developers in the documentation. 
+A tag that is more likely to be required to change:
+- DOXYGEN_EXCLUDE_PATTERNS: if you want to exclude certain source code directories or files from the documentation, you should add them here.

```

Comment:
Make this a section "ToDos".

Reply:
Resolved: https://github.com/tumcms/Open-Infra-Platform/pull/104/commits/80d5eb87c1bf79ad1969790ac663bf78e23e8f20